### PR TITLE
doc: Update documentation for GNSS assistance

### PR DESCRIPTION
**Pull request by tokangas**
Updated the nRF91 features documentation to use the term "A-GNSS" instead of "A-GPS".

Updated nrfxlib revision to pull in GNSS interface documentation update.

### CODE REVIEW

**Review by trantanen — APPROVED** [review]

File: doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst [hunk 1]
```diff
@@ -354,9 +357,9 @@ Samples using GNSS in |NCS|
...
-* :ref:`serial_lte_modem` application - Uses AT commands to start and stop GNSS and has support for nRF Cloud A-GPS and P-GPS. The application displays tracking and GNSS fix information in the serial console.
-* :ref:`gnss_sample` sample - Does not use assistance by default but can be configured to use nRF Cloud A-GPS or P-GPS or a combination of both. The sample displays tracking and fix information as well as NMEA strings in the serial console.
+* :ref:`asset_tracker_v2` application - Uses nRF Cloud for A-GNSS or P-GPS or a combination of both. The application obtains GNSS fixes and transmits them to nRF Cloud along with sensor data.
+* :ref:`serial_lte_modem` application - Uses AT commands to start and stop GNSS and has support for nRF Cloud A-GNSS and P-GPS. The application displays tracking and GNSS fix information in the serial console.
+* :ref:`gnss_sample` sample - Does not use assistance by default but can be configured to use nRF Cloud A-GNSS or P-GPS or a combination of both. The sample displays tracking and fix information as well as NMEA strings in the serial console.
```
Comment (peknis) [inline doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst:362]:
```suggestion
* The :ref:`asset_tracker_v2` application uses nRF Cloud for A-GNSS or P-GPS or a combination of both.
  The application obtains GNSS fixes and transmits them to nRF Cloud along with sensor data.

* The :ref:`serial_lte_modem` application uses AT commands to start and stop GNSS and has support for nRF Cloud A-GNSS and P-GPS.
  The application displays tracking and GNSS fix information in the serial console.

* The :ref:`gnss_sample` sample does not use assistance by default but can be configured to use nRF Cloud A-GNSS or P-GPS or a combination of both.
  The sample displays tracking and fix information as well as NMEA strings in the serial console.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst [hunk 2]
```diff
@@ -302,28 +302,30 @@ Enhancements to GNSS
...
-Assisted GPS (A-GPS)
----------------------
+Assisted GNSS (A-GNSS)
+----------------------
 
-A-GPS is commonly used to improve the Time to first fix (TTFF) by utilizing a connection (for example, over cellular) to the internet to retrieve the Almanac and Ephemeris data.
-A connection to an internet server that has the Almanac and Ephemeris data is several times quicker than using the slow 50 bps data link to the GPS satellites.
-There are many options to retrieve this A-GPS data.
+A-GNSS is commonly used to improve the Time to first fix (TTFF) by utilizing a connection (for example, over cellular) to the internet to retrieve the Almanac and Ephemeris data.
+A connection to an internet server that has the Almanac and Ephemeris data is several times quicker than using the slow 50 bps data link to the GNSS satellites.
```
[the remark this answers is not in the document]
Comment (tokangas) [reply doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst:312]: No, no need to capitalize those.

File: doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst [hunk 3]
```diff
@@ -332,13 +334,14 @@ Hence, the GNSS module needs to download the Ephemeris data from the satellite b
...
-------------------------------------------
+nRF Cloud compared with SUPL library
+------------------------------------
 
-The :ref:`lib_nrf_cloud_agps` library is more efficient to use when compared to the :ref:`SUPL <supl_client>` library, and the latter takes a bit more memory on the device.
-Another advantage of nRF Cloud A-GPS library is that the data is encrypted whereas SUPL uses plain socket.
-Also, no licenses are required from external vendors to use nRF Cloud A-GPS, whereas for commercial use of SUPL, you must obtain a license.
-The :ref:`lib_nrf_cloud_agps` library is also highly integrated into `Nordic Semiconductor's IoT cloud platform`_.
+* The :ref:`lib_nrf_cloud_agps` library is more efficient to use when compared to the :ref:`SUPL <supl_client>` library, and the latter takes a bit more memory on the device.
+* With nRF Cloud, the data is encrypted whereas SUPL uses plain socket.
```
Comment (melwee01) [inline doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst:341]:
```suggestion
* With nRF Cloud, the data is encrypted, whereas SUPL uses plain socket.
```

File: doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst [hunk 4]
```diff
@@ -332,13 +334,14 @@ Hence, the GNSS module needs to download the Ephemeris data from the satellite b
...
+* nRF Cloud supports assistance also for QZSS satellites, SUPL only for GPS.
```
Comment (melwee01) [inline doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst:342]:
```suggestion
* nRF Cloud also supports assistance for QZSS satellites. SUPL applies only to GPS.
```

File: doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst [hunk 5]
```diff
@@ -354,9 +357,12 @@ Samples using GNSS in |NCS|
 There are many examples in |NCS| that use GNSS.
 Following is a list of the samples and applications with some information about the GNSS usage:
 
-* :ref:`asset_tracker_v2` application - Uses nRF Cloud for A-GPS or P-GPS or a combination of both. The application obtains GNSS fixes and transmits them to nRF Cloud along with sensor data.
-* :ref:`serial_lte_modem` application - Uses AT commands to start and stop GNSS and has support for nRF Cloud A-GPS and P-GPS. The application displays tracking and GNSS fix information in the serial console.
-* :ref:`gnss_sample` sample - Does not use assistance by default but can be configured to use nRF Cloud A-GPS or P-GPS or a combination of both. The sample displays tracking and fix information as well as NMEA strings in the serial console.
+* The :ref:`asset_tracker_v2` application uses nRF Cloud for A-GNSS or P-GPS or a combination of both.
```
Comment (melwee01) [inline doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst:360]:
```suggestion
* The :ref:`asset_tracker_v2` application uses nRF Cloud for A-GNSS, P-GPS, or a combination of both.
```

File: doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst [hunk 6]
```diff
@@ -354,9 +357,12 @@ Samples using GNSS in |NCS|
...
+  The application obtains GNSS fixes and transmits them to nRF Cloud along with sensor data.
+* The :ref:`serial_lte_modem` application uses AT commands to start and stop GNSS and has support for nRF Cloud A-GNSS and P-GPS.
+  The application displays tracking and GNSS fix information in the serial console.
+* The :ref:`gnss_sample` sample does not use assistance by default but can be configured to use nRF Cloud A-GNSS or P-GPS or a combination of both.
```
Comment (melwee01) [inline doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst:364]:
```suggestion
* The :ref:`gnss_sample` sample does not use assistance by default but can be configured to use nRF Cloud A-GNSS, P-GPS, or a combination of both.
```

**Review by melwee01 — COMMENTED** [review]
A few minor syntax suggestions.

File: doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst (hunk 4 above)
Comment (tokangas) [reply doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst:342]: As discussed, I'll make it "nRF Cloud also support assistance for QZSS satellites, while SUPL is limited to GPS".

**Review by melwee01 — APPROVED** [review]

File: doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst [hunk 7]
```diff
@@ -302,43 +302,46 @@ Enhancements to GNSS
...
+nRF Cloud compared with SUPL library
+------------------------------------
 
-The :ref:`lib_nrf_cloud_agps` library is more efficient to use when compared to the :ref:`SUPL <supl_client>` library, and the latter takes a bit more memory on the device.
-Another advantage of nRF Cloud A-GPS library is that the data is encrypted whereas SUPL uses plain socket.
-Also, no licenses are required from external vendors to use nRF Cloud A-GPS, whereas for commercial use of SUPL, you must obtain a license.
-The :ref:`lib_nrf_cloud_agps` library is also highly integrated into `Nordic Semiconductor's IoT cloud platform`_.
+* The :ref:`lib_nrf_cloud_agps` library is more efficient to use when compared to the :ref:`SUPL <supl_client>` library, and the latter takes a bit more memory on the device.
+* With nRF Cloud, the data is encrypted, whereas SUPL uses plain socket.
+* nRF Cloud also support assistance for QZSS satellites, while SUPL is limited to GPS.
```
Comment (melwee01) [inline doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst:342]:
```suggestion
* nRF Cloud also supports assistance for QZSS satellites, while SUPL is limited to GPS.
```

File: doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst [hunk 8]
```diff
@@ -354,9 +357,12 @@ Samples using GNSS in |NCS|
 There are many examples in |NCS| that use GNSS.
 Following is a list of the samples and applications with some information about the GNSS usage:
 
-* :ref:`asset_tracker_v2` application - Uses nRF Cloud for A-GPS or P-GPS or a combination of both. The application obtains GNSS fixes and transmits them to nRF Cloud along with sensor data.
-* :ref:`serial_lte_modem` application - Uses AT commands to start and stop GNSS and has support for nRF Cloud A-GPS and P-GPS. The application displays tracking and GNSS fix information in the serial console.
-* :ref:`gnss_sample` sample - Does not use assistance by default but can be configured to use nRF Cloud A-GPS or P-GPS or a combination of both. The sample displays tracking and fix information as well as NMEA strings in the serial console.
+* The :ref:`asset_tracker_v2` application uses nRF Cloud for A-GNSS, P-GPS, or a combination of both.
+  The application obtains GNSS fixes and transmits them to nRF Cloud along with sensor data.
+* The :ref:`serial_lte_modem` application uses AT commands to start and stop GNSS and has support for nRF Cloud A-GNSS and P-GPS.
```
Comment (peknis) [inline doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst:362]:
```suggestion
* The :ref:`serial_lte_modem` application uses AT commands to start and stop GNSS and supports nRF Cloud A-GNSS and P-GPS.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

File: west.yml [hunk 9]
```diff
@@ -141,7 +141,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 5725f19f53c3fbdaa0ee0c079a8fecc1a5b78ae1
+      revision: pull/1080/head
```
Comment (rlubos) [inline west.yml:144]: Please update SHA: `5d8f0dc361db12814038a63c8a2f31306815ad2e`

**Pull request merged**